# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextCGL.h
# Commit: 0067bb1fb8e4
# Full Hash: 0067bb1fb8e45078f4383c9e371dd1c7eafbffbe
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2020-07-29 09:49:32
# Regressor Bug: 1652972
# File Overlap Count: 1
# Description:
#   Bug 1652972 - Mark EGL context as lost on Shutdown() r=aosmond
#   
#   This is meant to save us in cases where the message loop in GPU process
#   receives commands related to resources that point to the old EGL context
#   that was just shut down. Since the symbols are erased, we'd end up with
# ==============================================================================

diff -r bd33c30ba4b5 -r 0067bb1fb8e4 gfx/gl/GLContextCGL.h
--- a/gfx/gl/GLContextCGL.h	Tue Jul 28 22:26:21 2020 +0000
+++ b/gfx/gl/GLContextCGL.h	Wed Jul 29 02:33:54 2020 +0000
@@ -61,6 +61,8 @@
   // the active GPU has changed since the last call.
   void MigrateToActiveGPU();
 
+  virtual bool IsAliveImpl() const override;
+
   virtual bool MakeCurrentImpl() const override;
 
   virtual bool IsCurrentImpl() const override;